src/app/accounts/active-accounts.ts

```diff
@@ -0,0 +1,38 @@
+import { MS_PER_30_DAYS, USER_ACTIVENESS_MONTHLY_VOLUME_THRESHOLD } from "@config/app"
+import { LedgerService } from "@services/ledger"
+import { AccountsRepository } from "@services/mongoose"
+
+// user is considered active if there has been one transaction of more than USER_ACTIVENESS_MONTHLY_VOLUME_THRESHOLD sats in the last 30 days
```

Comment:
Is this explanation documented in form of a test?

Reply:
there are integration tests related to this, but I guess you're thinking of a unit test?